website/src/pages/documents/document.page.tsx

```diff
@@ -193,12 +248,14 @@ const DocumentContents = ({
   openDetails,
   tagSet,
   phoneticRepresentation,
+  wordPanelInfo,
 }: {
   doc: Document
   experienceLevel: ViewMode
   tagSet: TagSet
   openDetails: (morpheme: any) => void
   phoneticRepresentation: PhoneticRepresentation
+  wordPanelInfo: wordPanelDetails
```

Comment:
I wasn't sure how else I could pass on the information on the Word Panel, since the word panel itself is on the translation tab, and the segments I want to get it passed to are in the DocumentContents in the translation tab. It seemed easier to pass it down in an object than to pass down each thing individually.

Reply:
Yeah my feeling is if we are (almost) always passing these fields together and there are more than 2, then a sub-object makes sense. We can keep it for now at least.